components/price.js

```diff
@@ -80,6 +87,14 @@ export default function Price ({ className }) {
     )
   }
 
+  if (asSats === 'blockHeight') {
+    return (
+      <div className={compClassName} onClick={handleClick} variant='link'>
+        {numWithUnits(blockHeight, { abbreviate: false, unitSingular: 'block', unitPlural: 'blocks' })}
```

Comment:
I don't think you need `numWithUnits`. We will never reach 1 block again :)

I would have used `block 807175` [like in the block explorer](https://mempool.space/block/000000000000000000026b7f47d18638c29e9bbdb84fa4652f4332bead8c5244) but no strong opinions on that or using `numWithUnits`.

Reply:
> I don't think you need `numWithUnits`. We will never reach block 1 again :)

lol yea, I was just doing it for completeness, but you're right, it's not necessary.

> I would have used `block 807175` [like in the block explorer](https://mempool.space/block/000000000000000000026b7f47d18638c29e9bbdb84fa4652f4332bead8c5244) but no strong opinions on that or using `numWithUnits`.

I like that, too. I'll refactor to `block n`